omicron/cli/merge_with_gaps.py

```diff
@@ -145,7 +145,7 @@ def valid_file(path, uint_bug):
             table = EventTable.read(path, path='/triggers')
         elif path.name.endswith('.xml.gz') or path.name.endswith('.xml'):
             if uint_bug:
-                sed_cmd = ['sed', '-i', '-e', 's/uint_8s/int_8u/g', str(path.absolute())]
+                sed_cmd = ['sed', '-i', '', '-e', 's/uint_8s/int_8u/g', str(path.absolute())]
```

Comment:
Can you double-check this? Our quick test didn't find the problematic behaviour, but maybe this just needs another look

Reply:
Joe confirmed this behaviour. Possibly it is due to an odd behaviour in the `subprocess` package. We'll need to be attentive in case this behaviour changes in the future.